enterprise/admin/utils.py

```diff
@@ -111,4 +111,18 @@ def validate_email_to_link(email, raw_email=None, message_template=None, ignore_
         raise ValidationError(ValidationMessages.USER_ALREADY_REGISTERED.format(
             email=email, ec_name=existing_record.enterprise_customer.name
         ))
-    return bool(existing_record)
+    return existing_record or False
+
+
+def split_usernames_and_emails(email_field):
+    """
+    Split the contents of the email field into a list.
+
+    In some cases, a user could enter a comma-separated value inline in the
+    Manage Learners form. We should check to see if that's the case, and
+    provide a list of email addresses or usernames if it is.
+    """
+    from logging import getLogger
```

Comment:
Is this logger going to be removed prior to merging?  I'm not seeing a need for recording the email field in the server logs other than for debugging purposes?